ga.rpmtw.rpmlauncher.yml

```diff
@@ -0,0 +1,46 @@
+app-id: ga.rpmtw.rpmlauncher
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: RPMLauncher
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save config files
+  - --filesystem=home
+  # device access
+  - --device=dri
+modules:
+  - name: RPMLauncher
+    buildsystem: simple
+    only-arches:
+      - x86_64
+      - aarch64
+    build-commands:
+      - ls rpmlauncher
+      - cp -r rpmlauncher /app/rpmlauncher
+      - chmod +x /app/rpmlauncher/RPMLauncher
+      - install -Dm644 ga.rpmtw.rpmlauncher.png /app/share/icons/hicolor/scalable/apps/ga.rpmtw.rpmlauncher.png
+      - mkdir /app/bin
+      - ln -s /app/rpmlauncher/RPMLauncher /app/bin/RPMLauncher
+      - install -Dm644 ga.rpmtw.rpmlauncher.desktop /app/share/applications/ga.rpmtw.rpmlauncher.desktop
+      - install -Dm644 ga.rpmtw.rpmlauncher.metainfo.xml -t /app/share/metainfo
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
+          - aarch64
```

Comment:
```suggestion
```

Reply:
This is actually incorrect. The binary only contains `x86_64`.

But all you need is to restrict the architecture in the flathub.json

On the other hand if you can build from source, this would be better.